### PR TITLE
Updated some german strings

### PR DESCRIPTION
**Pull request by DAmNRelentless**
Only the first ones because I have no more time. I will continue asap.

### CODE REVIEW

**Comment by AirwavesMan** [comment]
Most of the string changes are not necessary. We have to stop here from changing everything again and again. A few strings could be changed but the rest is fine. Please dont change the strings from earlier translations if they are not wrong. I will mark the strings that are ok,.

File: SQF/dayz_code/stringtable.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		<Key ID="STR_ACTIONS_PUTOUTFIRE">
 			<English>Put out fire</English>
 			<Russian>Погасить огонь</Russian>
-			<German>Feuer ausmachen</German>
+			<German>Feuer löschen</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 2]
```diff
@@ -584,14 +584,14 @@
 			<Russian>Набор для переливания</Russian>
 			<French>Kit de transfusion</French>
 			<Czech>Transfúzní souprava</Czech>
-			<German>Transfusionsbesteck</German>
+			<German>Transfusionskit</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 3]
```diff
@@ -584,14 +584,14 @@
...
 		</Key>
 		<Key ID="STR_BLD_desc_transfusionKit">
 			<English>A blood transfusion kit that is required to perform blood transfusions and fill blood bags</English>
 			<Russian>Медицинский набор, необходимый для переливания крови.</Russian>
 			<French>Un kit de transfusion sanguine nécessaire pour transfuser un survivant ou remplir une poche de sang</French>
 			<Czech>Jednorázová transfúzní souprava, jež je nutná při provádění krevních transfúzí a plnění krevních vaků.</Czech>
-			<German>Ein Transfusionsbesteck, um Bluttransfusionen durchzuführen und Blutkonserven zu füllen.</German>
+			<German>Ein Transfusionskit, um Bluttransfusionen durchzuführen und Blutkonserven zu füllen.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 4]
```diff
@@ -612,14 +612,14 @@
 			<Russian>Еще ненаполненный пакет для крови</Russian>
 			<French>Une poche de sang vide à remplir avec un kit de transfusion</French>
 			<Czech>Prázdný krevní vak, jenž je možné naplnit vlastní krví s pomocí transfúzní soupravy.</Czech>
-			<German>Eine sterile leere Blutkonserve, die mit Blut befüllt werden muss.</German>
+			<German>Eine sterile leere Blutkonserve, die mit Blut befüllt werden kann.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 5]
```diff
@@ -612,14 +612,14 @@
...
 		</Key>
 		<Key ID="str_actions_medical_bagMissingKit">
 			<English>Missing Transfusion kit.</English>
 			<Russian>Отсутствует набор для переливания.</Russian>
 			<French>Kit de transfusion manquant</French>
 			<Czech>Chybí transfúzní souprava.</Czech>
-			<German>Transfusionsbesteck fehlt.</German>
+			<German>Transfusionskit fehlt.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 6]
```diff
@@ -779,7 +779,7 @@
 		</Key>
 		<Key ID="str_actions_medical_transfusion_failed_transfusionkit">
 			<English>Transfusion failed. Missing Transfusion Kit!</English>
-			<German>Transfusion fehlgeschlagen! Transfusionsbesteck fehlt!</German>
+			<German>Transfusion fehlgeschlagen! Transfusionskit fehlt!</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 7]
```diff
@@ -872,6 +872,7 @@
 		</Key>
 		<Key ID="str_actions_push">
 			<English>Push %1</English>
+			<German>%1 schieben</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 8]
```diff
@@ -1010,7 +1012,7 @@
 		</Key>
 		<Key ID="STR_ACTIONS_FILL_W">
 			<English>Fill Water</English>
-			<German>Mit Wasser füllen</German>
+			<German>Wasser abfüllen</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 9]
```diff
@@ -1258,7 +1260,7 @@
 			<Spanish>Ya estás pescando.</Spanish>
 			<Czech>Rybaření již probíhá.</Czech>
 			<French>Vous êtes déjà en train de pêcher</French>
-			<German>Du angelst schon.</German>
+			<German>Du angelst bereits.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 10]
```diff
@@ -1542,7 +1544,7 @@
 		<Key ID="str_sharpen_missing_water">
 			<English>You need a water bottle to sharpen objects.</English>
 			<Russian>Нужна вода, чтобы заточить объект.</Russian>
-			<German>Du brauchst eine Wasserflasche um Objekte zu schärfen.</German>
+			<German>Du brauchst eine Wasserflasche, um Objekte zu schärfen.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 11]
```diff
@@ -1573,15 +1575,15 @@
...
+			<German>Du kannst das nicht machen solange du auf einer Leiter oder im Wasser bist.</German>
 		</Key>
 
 		<Key ID="str_already_building">
 			<English>Already building</English>
 			<Russian>Объект уже строться.</Russian>
 			<French>La construction est déjà en cours</French>
 			<Czech>Stavba již probíhá.</Czech>
-			<German>Du baust schon.</German>
+			<German>Du baust bereits.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 12]
```diff
@@ -2087,7 +2089,7 @@
 			<Russian>Металлический прут, который может пригодиться в мастерской.</Russian>
 			<Czech>Kovová tyč se používá při výrobě.</Czech>
 			<French>Perche métallique (Artisanat)</French>
-			<German>Eine Metallstange. Kann beim Handwerken verwendet werden.</German>
+			<German>Eine Metallstange, die beim Handwerken verwendet wird.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 13]
```diff
@@ -2103,7 +2105,7 @@
 			<Russian>6 палаточных распорок, которые могут пригодиться в мастерской для создания палаток и масксетей.</Russian>
 			<Czech>Šest stanových tyčí, jež lze využít při výrobě stanů a kamuflážních sítí.</Czech>
 			<French>Perche métallique (Artisanat)</French>
-			<German>Sechs Zeltstangen aus Metall. Können beim Handwerken verwendet werden.</German>
+			<German>Sechs Zeltstangen aus Metall, die beim Handwerken verwendet werden.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 14]
```diff
@@ -2119,7 +2121,7 @@
 			<Russian>Резиновая трубка, которая может пригодиться в мастерской.</Russian>
 			<Czech>Gumová hadice se používá při výrobě.</Czech>
 			<French>Tuyau flexible (Artisanat)</French>
-			<German>Ein kurzer Gummischlauch. Kann beim Handwerken verwendet werden.</German>
+			<German>Ein kurzer Gummischlauch, der beim Handwerken verwendet wird.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 15]
```diff
@@ -2135,7 +2137,7 @@
 			<Russian>Фитиль, который может пригодиться в мастерской.</Russian>
 			<Czech>Knot se používá při výrobě.</Czech>
 			<French>Mèche (Artisanat)</French>
-			<German>Eine Lunte. Kann beim Handwerken verwendet werden.</German>
+			<German>Eine Lunte, die beim Handwerken verwendet wird.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 16]
```diff
@@ -2151,7 +2153,7 @@
 			<Russian>Перья, которые могут пригодиться в мастерской.</Russian>
 			<Czech>Peří se používá při výrobě. Nebo na lechtání.</Czech>
 			<French>Plumes (Artisanat)</French>
-			<German>Vogelfedern. Können beim Handwerken verwendet werden.</German>
+			<German>Vogelfedern, die beim Handwerken verwendet werden.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 17]
```diff
@@ -2167,7 +2169,7 @@
 			<Russian>Заглушка, которая может пригодиться в мастерской.</Russian>
 			<Czech>Zátka se používá při výrobě.</Czech>
 			<French>Plumes (Artisanat)</French>
-			<German>Eine Rohrkappe. Kann beim Handwerken verwendet werden.</German>
+			<German>Eine Rohrkappe, die beim Handwerken verwendet werden.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 18]
```diff
@@ -2211,7 +2213,7 @@
 		</Key>
 		<Key ID="STR_ITEM_DESC_equip_rope">
 			<English>Some rope</English>
-			<German>Ein Stück Seil.</German>
+			<German>Ein kurzes Seil</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 19]
```diff
@@ -2371,7 +2373,7 @@
 		</Key>
 		<Key ID="STR_ITEM_DESC_equip_metal_sheet">
 			<English>A clean piece of sheet metal</English>
-			<German>Eine Metallplatte. Kann beim Handwerken verwendet werden.</German>
+			<German>Eine Metallplatte, die beim Handwerken verwendet wird.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 20]
```diff
@@ -2307,7 +2309,7 @@
 		</Key>
 		<Key ID="STR_ITEM_DESC_equip_paint">
 			<English>Some paint</English>
-			<German>Etwas alte Farbe.</German>
+			<German>Etwas Farbe.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 21]
```diff
@@ -3373,7 +3377,7 @@
 		</Key>
 		<Key ID="STR_EQUIP_DESC_1b">
 			<English>A shovel is a tool for digging, lifting, and moving bulk materials.</English>
-			<German>Eine Schaufel. Man kann damit Löcher graben.</German>
+			<German>Eine Schaufel, mit welcher man Löcher graben kann.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 22]
```diff
@@ -3357,7 +3361,7 @@
 		</Key>
 		<Key ID="STR_EQUIP_DESC_1">
 			<English>Collapsible spade used together with the engineering backpack to create field defences</English>
-			<German>Ein Klappspaten. Wird beim Verschanzen und für viele Bautätigkeiten benötigt.</German>
+			<German>Ein Klappspaten, der beim Verschanzen und für viele Bautätigkeiten benötigt wird.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 23]
```diff
@@ -2680,7 +2683,7 @@
 			<Spanish>Bulbo del ajo, tiene cualidades medicinales puede ser utilizado para la elaboración de remedios caseros</Spanish>
 			<Czech>Hlavička česneku, kterou lze využít v alternativní medicíně. Nebo na upíry.</Czech>
 			<French>Gousse d&apos;ail, a des vertues médicinales et peut être utilisé pour créer des remèdes.</French>
-			<German>Eine Knolle Knoblauch, wegen seiner medizinischen Wirkung als Hausmittel bekannt.</German>
+			<German>Eine Knolle Knoblauch, welche wegen ihrer medizinischen Wirkung als Hausmittel bekannt ist.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 24]
```diff
@@ -2657,10 +2659,11 @@
 			<Spanish>Hojas de consuelda que pueden ser usadas para crear objetos.</Spanish>
 			<Czech>Kostival lékařský je léčivá bylina, jejíž listy jsou používány při výrobě.</Czech>
 			<French>Feuilles de consoude officinale (Artisanat)</French>
-			<German>Einige Blätter vom Rauen Beinwell. Wirken schmerzlindernd und entzündungshemmend.</German>
+			<German>Einige Blätter vom rauen Beinwell, welche schmerzlindernd und entzündungshemmend wirken.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 25]
```diff
@@ -2633,7 +2635,7 @@
 			<Russian>Напиток из смеси трав, который может помочь при инфекции.</Russian>
 			<Czech>Nápoj smíchá s bylinkami, které mohou pomoci léčit infekci.</Czech>
 			<French>Une boisson mélangée avec des herbes qui peuvent aider à guérir l'infection.</French>
-			<German>Ein Getränk aus Kräutern. Hilft bei Infektionen.</German>
+			<German>Ein Getränk aus Kräutern, welches bei Infektionen hilft.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 26]
```diff
@@ -2582,7 +2584,7 @@
 		</Key>
 		<Key ID="STR_ITEM_NAME_equip_d_battery">
 			<English>D Battery</English>
-			<German>Batterie D</German>
+			<German>D-Batterie</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 27]
```diff
@@ -2566,7 +2568,7 @@
 		</Key>
 		<Key ID="STR_ITEM_NAME_equip_aa_battery">
 			<English>AA Battery</English>
-			<German>Batterie AA</German>
+			<German>AA-Batterie</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

File: SQF/dayz_code/stringtable.xml [hunk 28]
```diff
@@ -2437,7 +2439,7 @@
 			<Spanish>Un pedazo de manguera en buen estado que podría ser usada para crear objetos.</Spanish>
 			<French>Un bout de tuyau d&apos;arrosage en bon état (Artisanat)</French>
 			<Czech>Kus hadice v dobrém stavu. Předmět lze použít při výrobě.</Czech>
-			<German>Ein Stück gut erhaltener Schlauch. Kann beim Handwerken verwendet werden.</German>
+			<German>Ein Stück gut erhaltener Schlauch, der beim Handwerken verwendet wird.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Ok

**Comment by DAmNRelentless** [comment]
No more necessary? You must be kidding. The grammar is horrible and for some items there is no german translation. Why should I ignore that? That volunteer help.

**Comment by AirwavesMan** [comment]
Please read carefully. The grammer is not horrible. You can adjust always some strings. That does not mean, that we have to. Every volunteer help is good but we have to review everything again and again. That will cost time. As @icomrade said, we should stop from adding and changing stuff. If there is something missing it is fine.
You changed many strings that dont need to be changed. A few strings have no german translation because it is the same translation from english in german. For example your Bandit change. Another thing is the length of the words. I did not change a few because there were to long for a rightclick option or does not match with a few other items.
It is really good that you will help but keep it to wrong things and missing strings. Otherwise we will never come to an end here.

**Comment by ebaydayz** [comment]
Committed the suggested revisions. 0525763

AirwavesMan is correct. At some point we need to limit changes to important and bug fixes only or we are never going to finish here. We've been working on this for almost a year already. I have limited time and desire left to work on this. I'm guessing @icomrade feels the same way. So we should focus on wrapping up the remaining important things and get 1.0.6 released. After it gets some real multiplayer testing a hotfix will probably be needed.

**Comment by icomrade** [comment]
I agree with that. I've been finding it harder and harder to allocate time to this project over the past couple of months. I'm sure time will only become more scarce for everyone during the Holidays I think we should patch up whatever we can in the coming week(s) and release preferably before Thanksgiving (obviously we'd all agree to this). The best course of action is probably getting as many people to test the most crucial game-play aspects and report any feedback in this time. I've made my test server public and added the stock BE filters to it, so any script restrictions should be added to the script filters on this repo fairly quickly (provided more people play on it).

The mod seems fairly stable as is, so minimal changes should be needed.

**Comment by ndavalos** [comment]
Might get more people to test over thanksgiving than before.

**Comment by icomrade** [comment]
I was thinking that would probably be better used for server admins to update (considering many probably still haven't started). Either was I wouldn't want to release on thanksgiving, probably closet a couple days before or after.

**Comment by oiad** [comment]
is there anything specifically we want to test?

**Comment by DAmNRelentless** [comment]
Okay, I agree with that and I also think that there will be more people in the future for testing especially on holidays. I will look trough the whole stringtable if anything german is missing and I only will add these strings to provide a full german translation. Keep on fixing bugs and concentrate on the important stuff. It's not necessary to approve all pull requests instantly.

**Comment by icomrade** [comment]
Like I said, I think we are pretty solid, at this point it may come down to finding most remaining issues after release. Playing the game normally with or without friends would probably be best to verify the script filter is good enough, it seems pretty permissive already and I haven't been kicked yet.

**Pull request closed without merging**